source/localizable/smart_cart/_order_object.html.md.erb

```diff
@@ -23,6 +23,7 @@ Name         | Type   | Value | Description
 `fulfilled_by_skroutz` | Boolean | | Whether the order is fulfilled by skroutz
 `fbs_delivery_note` | String | | Order's delivery note from warehouse for orders that are fulfilled by Skroutz
 `store_pickup` | Boolean | | Whether the order will be picked up by the user at the store
+`shipping_cost` | Double | | The order's shipping cost in euros (only for Fulfilled by Merchant orders, `0` for all other orders)
```

Comment:
i think we changed that

Reply:
Yes, thanks! Fixed and force pushed again.